lm_eval/tasks/arabic_leaderboard_complete/README.md

```diff
@@ -0,0 +1,254 @@
+      * `arabic_mmlu_high_school_statistics`
+      * `arabic_mmlu_high_school_us_history`
+      * `arabic_mmlu_high_school_world_history`
+      * `arabic_mmlu_human_aging`
+      * `arabic_mmlu_human_sexuality`
+      * `arabic_mmlu_international_law`
+      * `arabic_mmlu_jurisprudence`
+      * `arabic_mmlu_logical_fallacies`
+      * `arabic_mmlu_machine_learning`
+      * `arabic_mmlu_management`
+      * `arabic_mmlu_marketing`
+      * `arabic_mmlu_medical_genetics`
+      * `arabic_mmlu_miscellaneous`
+      * `arabic_mmlu_moral_disputes`
+      * `arabic_mmlu_moral_scenarios`
+      * `arabic_mmlu_nutrition`
+      * `arabic_mmlu_philosophy`
+      * `arabic_mmlu_prehistory`
+      * `arabic_mmlu_professional_accounting`
+      * `arabic_mmlu_professional_law`
+      * `arabic_mmlu_professional_medicine`
+      * `arabic_mmlu_professional_psychology`
+      * `arabic_mmlu_public_relations`
+      * `arabic_mmlu_security_studies`
+      * `arabic_mmlu_sociology`
+      * `arabic_mmlu_us_foreign_policy`
+      * `arabic_mmlu_virology`
+      * `arabic_mmlu_world_religions`
+* `arabic_openbook_qa`: A multiple-choice openbook question answering dataset that requires external knowledge and reasoning. The open book that comes with these questions is
+  based on elementary level science facts. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_piqa`: Physical Interaction Question Answering (PIQA) is a multiple-choice question answering based on physical commonsense reasoning. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_race`: A multiple-choice questions dataset to assess reading comprehension tasks based on English exams in China - designed for middle school and high school students
+  (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_sciq`: A multiple-choice Science Question Answering task to assess understanding of scientific concepts about physics, chemistry, and biology. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_toxigen`: This benchmark consists of tasks designed to evaluate language models and classify input text as hateful or not hateful. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `acva`: Arabic-Culture-Value-Alignment (ACVA) is a yes/no question dataset, generated by GPT3.5 Turbo from Arabic topics to assess model alignment with Arabic values and cultures.
+    * Paper: https://arxiv.org/pdf/2309.12053
+    * You can find the list of the tasks as follows:
+      - `arabic_Algeria`
```

Comment:
```suggestion
      - `acva_Algeria`
```
These task names are not descriptive enough--how should users know they are part of acva? could we change the names?

Reply:
added `acva` to the name of the related tasks for both the light and full versions

Reply:
perfect! Let's also add `arabic_leaderboard_...` to prefix to these task names, unless you disagree?